course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -83,6 +84,7 @@
                         roleName: $scope.getRoleName(user.role_id),
                     });
                 });
+            $scope.searchInProgress = false;
```

Comment:
yeah, looking at the possible ways the network call can end, I think we want to set this when either the get is successful or fails (post succeeded), or the post fails? sound right?